docs/streams/tutorial.html

```diff
@@ -485,12 +485,12 @@
                        return value;
                    }
                 })
-              .count("Counts");
+              .count(Materialized.&lt;String, Long, KeyValueStore&lt;Bytes, byte[]&gt;&gt; as("Counts"));
```

Comment:
Should we explain what `Bytes`is, and why we need it here?  I'd argue that most users get the use of `byte[]` but not `Bytes`.

Reply:
Small suggestion (based on user feedback where they shared confusion): let's rename "Counts" to "counts-store".

One example feedback I got was: "Do we need to use a string that has the same name as the operator, but make the string the plural form?"